ivy-openldap/users.ldif

```diff
@@ -12,6 +12,7 @@ version: 1
 # Entry 1: cn=rwei,dc=example,dc=org
 dn: cn=rwei,dc=example,dc=org
 cn: rwei
+uid: 000

```

Comment:
it's now required that the external identity provider is providing an ID of an user
otherwise it will not work

because isExternal is now checking only for externalId